XTRegTestSTATA1000.doc

```diff
@@ -0,0 +1,26 @@
+{\rtf1\fs24
+\trowd\trgaph75\trleft-75\intbl\trqc\clbrdrt\brdrs\cellx2160\clbrdrt\brdrs\cellx3600\clbrdrt\brdrs\cellx5040\clbrdrt\brdrs\cellx6768\clbrdrt\brdrs\cellx8784
+\ql \cell\qc (1)\cell\qc (2)\cell\qc (3)\cell\qc (4)\cell\row
+\trowd\trgaph75\trleft-75\trqc\cellx2160\cellx3600\cellx5040\cellx6768\cellx8784
+\trowd\trgaph75\trleft-75\trqc\clbrdrb\brdrs\cellx2160\clbrdrb\brdrs\cellx3600\clbrdrb\brdrs\cellx5040\clbrdrb\brdrs\cellx6768\clbrdrb\brdrs\cellx8784
+\ql VARIABLES\cell\qc Xtreg\cell\qc Xtreg-R\cell\qc Xtreg-one C\cell\qc Xtreg-one C-R\cell\row
+\trowd\trgaph75\trleft-75\trqc\cellx2160\cellx3600\cellx5040\cellx6768\cellx8784
+\ql \cell\qc \cell\qc \cell\qc \cell\qc \cell\row
+\ql x1\cell\qc 0.995***\cell\qc 0.995***\cell\qc 0.992***\cell\qc 0.992***\cell\row
+\ql \cell\qc (0.00480)\cell\qc (0.00519)\cell\qc (0.00586)\cell\qc (0.00586)\cell\row
+\ql x2\cell\qc -1.002***\cell\qc -1.002***\cell\qc \cell\qc \cell\row
+\ql \cell\qc (0.00480)\cell\qc (0.00567)\cell\qc \cell\qc \cell\row
+\ql Constant\cell\qc 0.896***\cell\qc 0.896***\cell\qc 0.993***\cell\qc 0.993***\cell\row
+\ql \cell\qc (0.0479)\cell\qc (0.0486)\cell\qc (0.0584)\cell\qc (0.0584)\cell\row
+\ql \cell\qc \cell\qc \cell\qc \cell\qc \cell\row
+\ql Observations\cell\qc 89,958\cell\qc 89,958\cell\qc 89,958\cell\qc 89,958\cell\row
+\ql R-squared\cell\qc 0.548\cell\qc 0.548\cell\qc 0.326\cell\qc 0.326\cell\row
+\ql Number of hhid\cell\qc 1,000\cell\qc 1,000\cell\qc 1,000\cell\qc 1,000\cell\row
+\ql Group FE\cell\qc YES\cell\qc YES\cell\qc YES\cell\qc YES\cell\row
+\ql Time FE\cell\qc YES\cell\qc YES\cell\qc YES\cell\qc YES\cell\row
+\trowd\trgaph75\trleft-75\trqc\trbrdrb\brdrs\clbrdrb\brdrs\cellx2160\clbrdrb\brdrs\cellx3600\clbrdrb\brdrs\cellx5040\clbrdrb\brdrs\cellx6768\clbrdrb\brdrs\cellx8784
+\ql Running Time\cell\qc 6.266\cell\qc 3.768\cell\qc 3.240\cell\qc 3.092\cell\row
+\pard\qc
+Standard errors in parentheses\par
+*** p<0.01, ** p<0.05, * p<0.1\par
+}
```

Comment:
Let's not commit output. Let's only commit code.

You can edit your .gitignore file to avoid tracking file some file extensions.

Reply:
Ok. Sorry for that, I will now edit my .gitignore to ignore the outputs and not push them.